src/main/java/com/google/cloud/resourcemanager/LienInfo.java

```diff
@@ -0,0 +1,218 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.cloud.resourcemanager;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+
+import com.google.common.base.Function;
+import java.io.Serializable;
+import java.util.List;
+import java.util.Objects;
+
+public class LienInfo implements Serializable {
+
+  static final Function<com.google.api.services.cloudresourcemanager.model.Lien, LienInfo>
+      FROM_PB_FUNCTION =
+          new Function<com.google.api.services.cloudresourcemanager.model.Lien, LienInfo>() {
+            @Override
+            public LienInfo apply(com.google.api.services.cloudresourcemanager.model.Lien pb) {
+              return LienInfo.fromPb(pb);
+            }
+          };
+  static final Function<LienInfo, com.google.api.services.cloudresourcemanager.model.Lien>
+      TO_PB_FUNCTION =
+          new Function<LienInfo, com.google.api.services.cloudresourcemanager.model.Lien>() {
+            @Override
+            public com.google.api.services.cloudresourcemanager.model.Lien apply(
+                LienInfo tableInfo) {
+              return tableInfo.toPb();
+            }
+          };
+
+  private static final long serialVersionUID = 9148970963697734236L;
+
+  private String createTime;
+  private String name;
+  private String origin;
+  private String parent;
+  private String reason;
+  private List<String> restrictions;
```

Comment:
List<String> -> ImmutableList<String>

Reply:
I think here,we should go with List Instead of ImmutableList.